sig-node/charter.md

```diff
@@ -0,0 +1,74 @@
+# SIG Node Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Node is responsible for the components that support the controlled
+interactions between pods and host resources.  We manage the lifecycle of pods
+that are scheduled to a node.  We focus on enabling a broad set of workload
+types, including hardware and performance sensitive workloads.  We maintain
+isolation boundaries between pods on a node, as well as the pod and the host.  We
+aim to continuously improve node reliability.
+
+### In scope
+
+Link to SIG section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Kubelet related features (e.g. Pod lifecycle)
```

Comment:
The template says: list of what qualifies a piece of code, binary or service as falling into the scope of this SIG

So how about: Kubelet and its features

Reply:
What about:
* The Pod API and Pod behaviors (some fields are owned by other SIGs, but...)
* The Node API
* Node controller
?